wagtail/wagtailcore/management/commands/set_url_paths.py

```diff
@@ -1,17 +1,30 @@
 from __future__ import absolute_import, unicode_literals
 
-from django.core.management.base import NoArgsCommand
+
+# Django < 1.10
+try:
+    from django.core.management.base import NoArgsCommand as BaseCommand
+except ImportError:
+    from django.core.management.base import BaseCommand
+
```

Comment:
`BaseCommand` has always been available, so this try/except seems unnecessary, especially since you moved the main logic to `handle` anyway.